File: lat3_c++/Disk.cpp
#include<iostream>
#include<string>
using namespace std;

class Disk
{
private:
    string type;
    string capacity;
    int price;

public:
    // konstruktor
    Disk() {}
    Disk(string type, string capacity, int price){
        this->type = type;
        this->capacity = capacity;
        this->price = price;
    }

    void setType(string type)
    {
        this->type = type;
    }

    string getType()
    {
        return this->type;
    }

    void setCapacity(string capacity)
    {
        this->capacity = capacity;
    }

    string getCapacity()
    {
        return this->capacity;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    int getPrice()
    {
        return this->price;
    }

    void outputDisk()
    {
        cout << "Disk Type          : " << this->getType() << endl;
        cout << "Disk Capacity      : " << this->getCapacity() << endl;
        cout << "Disk Price         : " << this->getPrice() << endl;
    }

    // destructor
    ~Disk() {}
};

File: lat3_python/Disk.py
class Disk():

    __Type = ""
    __Capacity = ""
    __Price = ""

    def __init__(self, Type = "", Capacity = "", Price = ""):
        self.__Type = Type
        self.__Capacity = Capacity
        self.__Price = Price

    def setType(self, Type):
        self.__Type = Type

    def setCapacity(self, Capacity):
        self.__Capacity = Capacity

    def setPrice(self, Price):
        self.__Price = Price

    def getType(self):
        return self.__Type

    def getCapacity(self):
        return self.__Capacity

    def getPrice(self):
        return self.__Price

    def printDisk(self):
        print("Disk Type        : ", str(self.getType()))
        print("Disk Capacity    : ", str(self.getCapacity()))
        print("Disk Price       : ", str(self.getPrice()))